sdcm/tester.py

```diff
@@ -906,6 +907,12 @@ def prepare_kms_host(self) -> None:
             'key_provider': 'KmsKeyProviderFactory',
             'kms_host': kms_host,
         }
+
+        if SkipPerIssues("scylladb/scylla-enterprise#4246", self.params):
+            # until https://github.com/scylladb/scylla-enterprise/issues/4246 would be solved
+            # we should disable the enterprise file streaming feature
+            append_scylla_yaml['enable_file_stream'] = False
```

Comment:
Will this option work for any supported Scylla enterprise version?
2023.1 ?

Reply:
No need, older versions doesn't have this option, so they won't do anything.